app/assets/javascripts/components/widgets_box.js.jsx

```diff
@@ -44,14 +44,14 @@ var Widgets = React.createClass({
   }
 })
 
-var Widget = React.createClass({
```

Comment:
I wanted the delete and edit methods to belong to to a Widget class that is inherited by all other widget classes. Alas, it seems that React believes in reuse by composition not inheritance. I tried to use that model with WidgetActions, and I can't see the use for a generic Widget class.